examples/weather/shared/src/favorites/events.rs

```diff
@@ -0,0 +1,595 @@
+use crux_core::{render::render, Command};
+use crux_http::command::Http;
+use crux_kv::{command::KeyValue, error::KeyValueError};
+use serde::{Deserialize, Serialize};
+use serde_json;
+
+use crate::config::API_KEY;
+use crate::favorites::model::{Favorite, FavoritesState, FAVORITES_KEY};
+use crate::location::model::geocoding_response::{
+    GeocodingQueryString, GeocodingResponse, GEOCODING_URL,
+};
+
+use crate::{Effect, Workflow};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub enum FavoritesEvent {
+    // Workflow - Favorites view
+    DeletePressed(Box<Favorite>),
+    DeleteConfirmed,
+    DeleteCancelled,
```

Comment:
I wonder whether the confirmation dialog should be considered a "UI only concern" ... very debatable of course.